examples/bri-3/README.md

```diff
@@ -1,39 +1,39 @@
-<a href="https://discord.gg/G7Qnnhy" target="_blank"><img src="https://img.shields.io/badge/discord-online-brightgreen.svg" alt="Discord"/></a>
-<a href="https://opencollective.com/nest#backer" target="_blank"><img src="https://opencollective.com/nest/backers/badge.svg" alt="Backers on Open Collective" /></a>
-<a href="https://opencollective.com/nest#sponsor" target="_blank"><img src="https://opencollective.com/nest/sponsors/badge.svg" alt="Sponsors on Open Collective" /></a>
-  <a href="https://paypal.me/kamilmysliwiec" target="_blank"><img src="https://img.shields.io/badge/Donate-PayPal-ff3f59.svg"/></a>
-    <a href="https://opencollective.com/nest#sponsor"  target="_blank"><img src="https://img.shields.io/badge/Support%20us-Open%20Collective-41B883.svg" alt="Support us"></a>
-  <a href="https://twitter.com/nestframework" target="_blank"><img src="https://img.shields.io/twitter/follow/nestframework.svg?style=social&label=Follow"></a>
-</p>
-  <!--[![Backers on Open Collective](https://opencollective.com/nest/backers/badge.svg)](https://opencollective.com/nest#backer)
-  [![Sponsors on Open Collective](https://opencollective.com/nest/sponsors/badge.svg)](https://opencollective.com/nest#sponsor)-->
-
 ## Description
 
-[Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
+BRI-3 is a simple reference implemenation of the baseline standard, developed under [grant 85](https://github.com/eea-oasis/baseline-grants/issues/85).  
+
+
+This reference implementation is being built from scratch and will contain a limited set of features. It will serve the following purposes:
+
+ * Plug and play Baseline Reference Implementation that can cover a number of business use-cases in live environments
+
+ * Basis for community adoption and participation of companies with real-life business cases in extending the protocol
+
+ * Trying out the concepts from the standard in practice and producing feedback loops for standard improvements
+
+ * Entry point for developers to get to know the protocol, and start contributing in iterative and atomic fashion
+
+ * Basis for the Baseline SDK
+
+ * Set an example of how to build an open-source project under the Baseline protocol in regard to transparency, community participation and collaboration
+
+ * First vendor-agnostic implementation of the protocol.
+
+## Prerequisites
+
+[Docker] (https://docs.docker.com/engine/install/)
+
+[Node LTS] (https://nodejs.org/en/)
 
 ## Installation
 
 ```bash
-$ npm install
-$ npm install -g prisma
-$ prisma generate
-$ npx prisma migrate dev
-$ npx ttsc
+$ docker run --name postgres -e POSTGRES_PASSWORD=example -p 5432:5432 -d postgres # start a postgres container
+$ create a .env file based on the .env.sample # provide a connection string for the db instance
+$ npm install # install project dependencies
+$ npm install -g prisma # install prisma globally
+$ prisma generate # generate the prisma client 
+$ npx prisma migrate dev # migrate the db to latest state
```

Comment:
what about ttsc step?

Reply:
i didnt' need it locally to run the app + will be obsolete with automapper

Reply:
with every change in code you need to run npm build because app is now served from dist folder, but ok i guess we dont need it in readme